### ISSUE TITLE
Bug: Contextmenu layout not correct with icons

### ISSUE THREAD

**Issue by daniel-sc** [sbb-design-systems/sbb-angular#198]
**Describe the bug**
When using the contextmenu with buttons and include icons the alignment is not correct. 
Furthermore the icon is not changing color on hover.

![image](https://user-images.githubusercontent.com/117919/67267823-6312a380-f4b3-11e9-8ebf-794187bfc743.png)

**To Reproduce**
```
  <sbb-contextmenu>
    <sbb-dropdown>
      <button sbbDropdownItem>
        <sbb-icon-face-worker size="small"></sbb-icon-face-worker>
        Kopieren
      </button>
      <button sbbDropdownItem>
        <sbb-icon-trash size="small"></sbb-icon-trash>
        Löschen
      </button>
    </sbb-dropdown>
  </sbb-contextmenu>
```

**Expected behavior**
Horizontal alignment should be correct (middle) and icon should have red color on hover.

**Screenshots**
see above.

**Comment by kyubisation** [comment]
The positioning of the icons is not trivial due to them being SVGs. Does it help if you configure <button> to be display: flex?

As for hover, #152 might help in this regard.

**Comment by daniel-sc** [comment]
Hi @kyubisation,
it seems that adding
```css
display: flex;
align-items: center;
```
to the button and
```css
margin-right: 8px;
```
to the icon would fix the layout problems (at least in chrome).

**Comment by kyubisation** [comment]
Closed by #607
